internal/resources/providers/azurelib/inventory/susbcriptions_provider.go

```diff
@@ -54,14 +66,37 @@ func NewSubscriptionProvider(log *logp.Logger, credentials azcore.TokenCredentia
 		},
 	}
 
+	tenantClient := tenantAzureClientWrapper{
+		Tenants: func(ctx context.Context, clientOptions *arm.ClientOptions, options *armsubscriptions.TenantsClientListOptions) ([]armsubscriptions.TenantsClientListResponse, error) {
+			c, err := armsubscriptions.NewTenantsClient(credentials, clientOptions)
+			if err != nil {
+				return nil, err
+			}
+			return readPager(ctx, c.NewListPager(options))
+		},
+	}
+
+	subscriptionClient := subscriptionAzureClientWrapper{
+		Subscriptions: func(ctx context.Context, clientOptions *arm.ClientOptions, options *armsubscriptions.ClientListOptions) ([]armsubscriptions.ClientListResponse, error) {
+			cl, err := armsubscriptions.NewClient(credentials, clientOptions)
```

Comment:
Can't we get subscriptions from the graph api?

Reply:
ditto

I checked, but haven't found subscriptions in Resource Graph API.

Reply:
Same comment as to `tenants` I see a `microsoft.resources/subscriptions` resource, but without results nor context nor documentation 🫠